frontend/src/pages/AddLightsPage.js

```diff
@@ -8,31 +8,70 @@ import {useAuth} from "../auth/AuthProvider";
 import TextField from "@material-ui/core/TextField";
 import {ButtonGroup, CircularProgress} from "@material-ui/core";
 import {useState} from "react";
-import {connectHub} from "../services/hub-api-service";
+import {addLights} from "../services/lights-api-service";
+
+const initialState = {
+    deviceName: '',
+    uid: '',
+    itemName: ''
+}
 
 export default function AddLightsPage() {
+    const {token} = useAuth()
+    const [error, setError] = useState()
+    const [loading, setLoading] = useState(false)
+    const [credentials, setCredentials] = useState(initialState)
+    const [redirect ,setRedirect] = useState()
+
+    const handleCredentialsChange = event =>
+        setCredentials({...credentials,[event.target.name]: event.target.value})
+
+    const handleSubmit = event => {
+        event.preventDefault()
+        setError()
+        setLoading(true)
+        addLights(token, credentials).then(light => setRedirect(light.deviceName))
+            .catch( error => {
+                setError(error)
+                setLoading(false)
+                })
+    }
+
+    const handleClear = () => {
+        setCredentials({
+            deviceName: '',
+            uid: '',
+            itemName: ''
+        })
+    }
+
+    if(redirect) {
+        return <Redirect to ="/lights"/>
```

Comment:
sollte hier nich der wert von redirect in den to="/lights"  mit rein ?

Reply:
hey matthias. Ich habs so eingestellt dass bei submit sich der Wert von null auf irgendetwas ändert, und dann redirected wird. Der Wert ist mir eigentlich egal, weil in "/lights" sowieso alle Lichter über eine Get-Methode geholt werden. Ist also rein zum redirecten da